### ISSUE TITLE
project.properties duplicate conflict b/w google-cloud-core & google-cloud-pubsub

### ISSUE THREAD

**Issue by RaiYogesh** [GoogleCloudPlatform/google-cloud-java#2411]
I am using duplicate-finder-maven-plugin in my project which has started complaining for duplicate project.properties b/w  google-cloud-core & google-cloud-pubsub

`[INFO] Checking test classpath

[WARNING] Found duplicate and different resources in [com.google.cloud:google-cloud-core:1.2.1, com.google.cloud:google-cloud-pubsub:0.20.1-beta]:
[WARNING]   project.properties
[WARNING] Found duplicate classes/resources in compile classpath.
[WARNING] Found duplicate and different resources in [com.google.cloud:google-cloud-core:1.2.1, com.google.cloud:google-cloud-pubsub:0.20.1-beta]:
[WARNING]   project.properties
[WARNING] Found duplicate classes/resources in runtime classpath.
[WARNING] Found duplicate and different resources in [com.google.cloud:google-cloud-core:1.2.1, com.google.cloud:google-cloud-pubsub:0.20.1-beta]:
[WARNING]   project.properties
[WARNING] Found duplicate classes/resources in test classpath.`

Kindly help out

**Comment by jakub-bochenski** [comment]
I think better place for this `project.properties` would be under `META-INF`.

In fact version is stored there already in `MANIFEST.MF` and in Maven descriptor.

**Comment by michaelbausor** [comment]
I think this is something we should investigate - it may be that `META-INF` is better than what we do at the moment. I will file a separate issue to track that.

As a temporary workaround: I am not familiar with the plugin, but have you looked at using the `ignoredResourcePatterns` setting (seen on this page: https://github.com/basepom/duplicate-finder-maven-plugin/wiki). That might help unblock or resolve the warning in the short term.

**Comment by jakub-bochenski** [comment]
> I am not familiar with the plugin, but have you looked at using the ignoredResourcePatterns setting 
Yes, we do that already but it's bad practice to have conflicting entries on classpath anyway
```
                                <conflictingDependencies>
                                    <dependency>
                                        <groupId>com.google.cloud</groupId>
                                        <artifactId>google-cloud-core</artifactId>
                                    </dependency>
                                    <dependency>
                                        <groupId>com.google.cloud</groupId>
                                        <artifactId>google-cloud-pubsub</artifactId>
                                    </dependency>
                                </conflictingDependencies>
                                <resources>
                                    <resource>project.properties</resource>
                                </resources>
```

**Comment by danoscarmike** [comment]
Ping @michaelbausor - what are next steps here?

**Comment by michaelbausor** [comment]
@garrettjonesgoogle Are we planning on kicking off work to migrate from Maven to Gradle soon? (https://github.com/GoogleCloudPlatform/google-cloud-java/issues/1587) If so, this issue may be best handled as part of that transition.

**Comment by jakub-bochenski** [comment]
I just found this has been fixed in db0fdb18a562b1f11c06839fd1b2f677d197cb00#diff-c7d3811896974c0e04fd2160fb779ad0

I see it removed the offending sections from multiple poms, though the commit message makes no mention of it

**Comment by garrettjonesgoogle** [comment]
Correct, this is fixed now. Closing issue.
